src/limiters/limiters_dg1d.jl

```diff
@@ -0,0 +1,40 @@
+
+function limiter_zhang_shu!(u::AbstractArray{<:Any,3},
+                            threshold::Real, variable,
+                            mesh, equations, dg::DGSEM, cache)
+  @unpack weights = dg.basis
+
+  Threads.@threads for element in eachelement(dg, cache)
+    # dermine minimum value
+    value_min = typemax(eltype(u))
+    for i in eachnode(dg)
+      u_node = get_node_vars(u, equations, dg, i, element)
+      value_min = min(value_min, variable(u_node, equations))
+    end
+
+    # detect if limiting is necessary
+    value_min < threshold || continue
```

Comment:
```suggestion
    if value_min < threshold
      continue
    end
```

I prefer "full" if conditions for such control statements, as they are visually easier to recognize. But I also recognize this is a personal preference, so it's up to you if you adopt it.

Reply:
On the other hand, patterns like this are widespread in Julia. Hence, it can be beneficial to make people aware of them and use them also in Trixi.

Reply:
OK. As I said, I am OK either way.